### PR TITLE
feat: add pull request template

### PR DESCRIPTION
**Pull request by dreyfus92**
adds a standardized pull request template. this was heavily inspired by emdashes [pr template](https://github.com/emdash-cms/emdash/blob/main/.github/PULL_REQUEST_TEMPLATE.md) ✌🏻

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## What does this PR do?
+
+<!-- Describe the change and why it's needed. Link to a related issue or discussion. -->
```
Comment (43081j) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!--
  Describe the change and why it's needed. Link to a related issue or discussion.
  If there is no issue, please explain why one isn't needed.
-->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## AI-generated code disclosure
+
+<!-- If any part of this PR was generated by AI tools (Copilot, Claude, GPT, Cursor, etc.), check the box. This is fine — we just need to know so reviewers can pay extra attention to edge cases. -->
+
+- [ ] This PR includes AI-generated code
+
+## Screenshots / test output
+
+<!-- Optional. Include if the change is visual or if you want to show test results. -->
```
Comment (43081j) [inline .github/PULL_REQUEST_TEMPLATE.md]: we probably don't need this bit. if someone wants to show their change, it can go in the main description i think so it is paired with context

**Review by 43081j — APPROVED** [review]

**Comment by dreyfus92** [comment]
thank you @43081j ✌🏻
